tests/ci_build/Dockerfile.gpu

```diff
@@ -35,7 +35,7 @@ ENV CPP=/opt/rh/devtoolset-2/root/usr/bin/cpp
 
 # Install Python packages
 RUN \
-    pip install numpy pytest scipy scikit-learn wheel
+    pip install numpy pytest scipy scikit-learn wheel kubernetes urllib3==1.22
```

Comment:
Necessary additions to fix unsatisfiable package sequence errors. Kubernetes is required for the `dmlc_submit` script.